locales/templates/app.pot

```diff
@@ -531,15 +531,27 @@ msgctxt "firstrun_next_button"
 msgid "Next"
 msgstr ""
 
-#. The welcome titles in Firefox Lite 2.5
+#. !NOT USE! The welcome titles in Firefox Lite 2.5
 msgctxt "firstrun_fxlite_2.5_title_1"
 msgid "Hello!"
 msgstr ""
 
+#. !NOT USE! The welcome titles in Firefox Lite 2.5 for asking users to choose
```

Comment:
Nit: do we need this? "!NOT USE!"

Reply:
Hi Delphine, 
Yes we need this! 
When we have a string that needs an update on the string ID, we can't make any changes on the old string ID or there will be some risks on generating some bugs. Our current solution is to keep the old string ID and add "!NOT USE!" in the comment so that we can remove the old ones quarterly.

Reply:
ok thanks for explaining